include/deal.II/grid/tria_accessor.templates.h

```diff
@@ -688,7 +688,32 @@ namespace internal
       }
 
 
-      inline static unsigned int
+
+      template <int dim, int spacedim>
+      inline static unsigned char
+      combined_face_orientation(
+        const TriaAccessor<dim, spacedim, 1> & /*accessor*/,
+        const unsigned int /*face*/)
+      {
+        // There's only one way to orient a vertex
+        return ReferenceCell::default_combined_face_orientation();
+      }
```

Comment:
This does not look right. `TriaAccessor` is declared like this:
```
template <int structdim, int dim, int spacedim>
class TriaAccessor
```
So `dim` and `spacedim` are the second and third template argument, but you're using them as the first and second.

Reply:
Separately, a vertex is a zero-dimensional object, not a one-dimensional one.

Reply:
I let @drwells answer this one and the next one.